src/OrganizationsSDKAdapter.js

```diff
@@ -0,0 +1,70 @@
+import {ReplaySubject, defer} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {deconstructHydraId} from '@webex/common';
+import {OrganizationsAdapter} from '@webex/component-adapter-interfaces';
+
+// TODO: Figure out how to import JS Doc definitions and remove duplication.
+/**
+ * A Member object that is part of a membership
+ *
+ * @external Organization
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/OrganizationsAdapter.js#L6}
+ */
+
+/**
+ * The `OrganizationsSDKAdapter` is an implementation of the `OrganizationsAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a organization.
+ *
+ * @implements {OrganizationsAdapter}
+ */
+export default class OrganizationsSDKAdapter extends OrganizationsAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.organizationObservables = {};
+  }
+
+  /**
+   * Fetches the organization data and returns in the shape required by adapter.
```

Comment:
This method doesn't seem to be doing any shaping 😉 
```suggestion
   * Returns a promise to organization data from Webex.
```